### DISCUSSION TITLE
AI influencer

### DISCUSSION THREAD

**Discussion by monkrus** [lllyasviel/Fooocus#2036] [Show and tell]
The trigger for this topic was [#1329](https://github.com/lllyasviel/Fooocus/issues/1329). 
Let`s take a broader look at the development of AI influencers.

We could concentrate on:

1. **Settings and finetuning for image creation:**
   - What settings do you tweak to ensure impeccable image quality?
   - Can you share insights into the finetuning process you use for crafting visuals?
   - Any favorite combinations or secret sauce you've discovered for creating standout images?

1. **Original Concept:**
   - Original concept for your AI influencer? What inspired it?
   - Base model, how did you tailor it to be unique?
   
2. **Engagement on Social Media and Monetization:**
   - Defining your target audience. Any surprises along the way?
    -How do you monetize your AI influencer platform?
    -Any multiple revenue streams, like sponsored content, partnerships, or something unique?
    -Lessons learned in the monetization journey? Success stories or challenges faced?

3. **Network of Influencers/Agents:**
   - Do you collaborate with other AI influencers? How does that work?
   - Any funny or interesting stories from interactions with other virtual personalities?
   -Any thoughts on the future of this interconnected AI influencer community?

4. **Personalized/Custom-Made Influencer:**
   - Additional features are you thinking of adding? Impeccable image quality, videos, audio?
   - How do you make your influencer stand out as a personalized experience?
   - Challenges faced and victories achieved in creating a custom-made virtual personality?

**Comment by monkrus** [comment]
My humble experience so far: [Ingrid Hansen](https://www.instagram.com/norskingy/)
The base image was created using a long prompt in Leonardo AI.
The base idea is a 22-year-old Norwegian girl, traveler, and motivator.
I use default Style + Fooocus Masterpiece and Fooocus Photograph.
Eyes and hands are the most challenging.
Each image goes through a 2x upscaling and Improve Detail option.
Instagram and Fanvue are used as social platforms.
Needs more research on next steps, work in progress :)

**Reply by mashb1t** [reply]
So no LoRA and only using one image + one prompt for generation?

**Reply by monkrus** [reply]
Correct, with the finetuning steps I mentioned above. I will try your steps as well.

**Comment by mashb1t** [comment]
I've had very good experiences with LoRA generation using https://github.com/bmaltais/kohya_ss.
Not an AI influencer, but the basis for https://gallery.mashb1t.de/gallery/9OwaB8_NAtZaULpAE95_DK5g is a LoRA trained on the SDXL base model with 19 images of myself in various poses, mostly portraits.
With the help of Fooocus image prompts as reference the style can easier be applied.

Usual workflow:
1. select LoRA
2. upload references in image prompt tab
3. write prompt
4. generate
5. inpaint hands, generate again
6. upscale x2 with mix of prompt to keep the face trained in the LoRA

**Reply by monkrus** [reply]
@mashb1t  If I replace default LoRA with LORA of my choice in Fooocus (for more realistic images), would I apply the same weight? What is the rule of thumb here if I were to play with weights/multiple LoRAs?

**Reply by mashb1t** [reply]
Each LoRA works best within its specific weight range (see model card).
Rule of thumb: set LoRA weight to 1 if you plan to swap the sdxl offset LoRA with another one.

**Reply by monkrus** [reply]
Thank you! A bit of a random question, but still:  how can I "dress" my AI model into a  XYZ brand clothing. Is it doable? If yes, what are the steps?

**Reply by mashb1t** [reply]
This is not yet doable really good (only prompts), but check out Diffuse2Choose by Amazon: https://diffuse2choose.github.io/

**Reply by monkrus** [reply]
Seems like @mashb1t has answers to all questions. What technique would I use to "meet" to make my AI model meet someone else, e.g. If I have my base image and another base image, how would create an image where they are side by side :) ?

**Reply by mashb1t** [reply]
That's tricky, i assume the best way to currently do this is to create a background image and then do 2x inpainting, one for each person, using the result of the first as inpaint basis for the second.

**Reply by monkrus** [reply]
@mashb1t  Still no code is available for " dress your model"/DiffuseToChoose. Do you think it might be implemented in Fooocus? Any other current alternatives?

**Comment by monkrus** [comment]
I hate to ask this question again. Full height,  no cut-offs images. It is is a hit and miss to get the head to toe image. What are the best settings, resolution, style etc. to create a realistic full size image? Thanks!

**Reply by mashb1t** [reply]
1:2 or another tall aspect ratio for full human body pictures, then outpaint
OR
"full body, zoomed out shot" with higher prompt strength

**Reply by barepixels** [reply]
include feet or shoe in prompt

**Comment by sandrapisano** [comment]
Good morning I'm new she is my influencer that I would like to improve and a woman in her 40s https://www.instagram.com/sandrapisanomusic/ I use the basic functions I don't touch anything because I'm not experienced but I would like to improve them what commands do you recommend I use fooocus collab from the website sometimes the hands are not perfect and I would like to take full photos plus I would like to have another character in the photos with her that I created separately how can I put them in the same photo together? I apologize but I saw that you are very good and I would like some teaching to improve thank you very much.
